engines/m4/burger/rooms/section6/room602.cpp

```diff
@@ -1202,6 +1202,7 @@ void Room602::daemon() {
 			break;
 
 		case 42:
+			_G(wilbur_should) = 10002;
```

Comment:
It doesn't happen in the whole game, but I tested and it doesn't hurt it either, so this is fine.